jsonnet/lib/observatorium-api.libsonnet

```diff
@@ -190,6 +190,17 @@
                    if std.objectHas(tenant, 'mTLS')
                    if std.objectHas(tenant.mTLS, 'caKey')
                  ] else []) +
+                (if std.objectHas(api.config.tenants, 'tenants') then [
+                   {
+                     name: tenant.name + '-tls-configmap',
+                     mountPath: tenant.oidc.issuerCAPath,
+                     subPath: tenant.oidc.caKey,
+                     readOnly: true,
+                   }
+                   for tenant in api.config.tenants.tenants
+                   if std.objectHas(tenant, 'oidc')
+                   if std.objectHas(tenant.oidc, 'caKey')
```

Comment:
this assumes that if `caKey` is set, `issuerCAPath` is also set

Reply:
Ack this is a good assumption